docs/architecture/decisions/use-department-group-permissions-in-place-of-department-relation-models.md

```diff
@@ -0,0 +1,38 @@
+<!-- # Title --> 
+# Use department group permissions in place of related Department models
+
+## Status
+<!-- What is the status, such as proposed, accepted, rejected, deprecated, superseded, etc.? -->
+Accepted
```

Comment:
This is kinda ambiguous, but I figure since this PR is proposed as merged into the PR that will make this change, it might as well count as 'accepted'.